m4/ax_c_mallinfo.m4

```diff
@@ -0,0 +1,21 @@
+# AC_C_MALLINFO
+# -------------
+# Define HAVE_C_MALLINFO if mallinfo() works.
+AN_IDENTIFIER([mallinfo], [AC_C_MALLINFO])
+AC_DEFUN([AC_C_MALLINFO],
+[AC_CACHE_CHECK([for mallinfo], ac_cv_c_mallinfo,
+[AC_LINK_IFELSE(
+   [AC_LANG_SOURCE(
+      [[#include <malloc.h>
+        int main(int argc, char **argv) {
+          struct mallinfo mi = mallinfo();
+          printf("%d%d%d\n", mi.uordblks, mi.hblkhd, mi.arena);
```

Comment:
I needed to make sure `struct mallinfo` has the right members. Without it, an incompatible mallinfo will give a false positive match and netdata will break to be linked.

Right?